back-end/src/utilidades/tokenUtils.js

```diff
@@ -0,0 +1,22 @@
+const JWT = require('jsonwebtoken');
+
+const jwtSecret = process.env.JWT_SECRET;
+
+const gerarToken = (dados) => {
+  const token = JWT.sign(dados, jwtSecret);
+  return token;
+};
```

Comment:
```suggestion
const gerarToken = (dados) => sign(dados, jwtSecret);
```
Acho q fica mais clean 😎